examples/3_Advanced_Topics/3-2_Creating_an_XML_BoM.py

```diff
@@ -0,0 +1,247 @@
+﻿# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.15.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # Create an XML BoM using python
+
+# The `bom_types` namespace can be used in conjunction with the `BoMHandler` class to
+# create and manipulate BoMs for analysis. This example shows how this can be done, creating
+# a BoM for a laminated glass door and running a compliance query.
+
```

Comment:
Maybe give a brief description of a door at this point? Something like:

"The door contains two hinges and a handle, both fixed to the frame with machine screws and washers. The door glass is coated with a reflective polymer film".

That then gives you some context to refer back to when creating the BoM.